### DISCUSSION TITLE
A list of entries the form has been used on

### DISCUSSION THREAD

**Discussion by wolispace** [solspace/craft-freeform#1462] [Feature Requests]
It would be great if each form also shows which entries they are currently used on.

Another tab called 'Usage' and on that a list of each entry that includes it and if that entry is enabled or not.

**Comment by kjmartens** [comment]
Thank you for your feedback @wolispace. This is under consideration. 🙂

**Comment by kjmartens** [comment]
This is now available in Freeform 5.7.2+ (on Craft 5) 🙂
